empty-plugin/src/main/resources/archetype-resources/pom.xml

```diff
@@ -37,6 +37,7 @@
     <!-- https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/ -->
     <jenkins.baseline>2.479</jenkins.baseline>
     <jenkins.version>${jenkins.baseline}.3</jenkins.version>
+    <jenkins.bom.version>5054.v620b_5d2b_d5e6</jenkins.bom.version>
```

Comment:
I'd rather avoid extracting this to a version as its not reused, is it necessary?

Reply:
Renovate already updates it just fine? 

https://github.com/jenkinsci/archetypes/pull/826

Reply:
I was updating it to ensure that it is kept compatible with the version of Jenkins selected. If that isn't needed I can remove it to simplify things

Reply:
removed in f6eac055c24fe60c4820b11859988d1756b7dea5